.github/workflows/build.yml

```diff
@@ -3,8 +3,24 @@ name: Build
 on:
   push:
     branches: master
+    paths:
```

Comment:
Probably `misc\msi-installer` too?

Reply:
Ah, I forgot that we haven't migrated the the installer from appveyor.
Please ignore.